KK-DZ/src/app/core/models/employee-origin.ts

```diff
@@ -0,0 +1,8 @@
+import { Employee } from "./employee";
+
+export const EmployeeOrigin: Employee = {
```

Comment:
nigdy nie zaczynamy z dużej litery zmiennych (PascalCase)

stałe albo cała nazwa. z dużych liter lub camelCase

```suggestion
export const EMPLOYEE_ORIGIN: Employee = {
```